s3/pkg/meta/db/drivers/tidb/object.go

```diff
@@ -48,10 +48,10 @@ func (t *TidbClient) GetObject(ctx context.Context, bucketName, objectName, vers
 	err = row.Scan(
 		&ibucketname,
 		&iname,
-		&iversion,
+		&object.VersionId,
 		&object.Location,
-		&object.TenantId,
-		&object.UserId,
+		//&object.TenantId,
```

Comment:
Can you put some comment as to why we have removed tenant and userId from here, will this affect Multi-tenant support in multi-cloud?